ModPatches/Spacer Arsenal/Patches/Spacer Arsenal/Apparel_Various.xml

```diff
@@ -0,0 +1,97 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+
+	<!-- Body Armour -->
+
+	<Operation Class="PatchOperationAdd">
+		<xpath>Defs/ThingDef[defName="DV_Apparel_EnsignArmor"]</xpath>
```

Comment:
Rather than patching the fields into the child def, it's probably better to patch the parent def `Name=""DV_ApparelArmorEnsignBase"` directly. That way, it handles any variants of the armor that use the same parent (such as with cosmetic variants).